tests/acceptance/qunit-dom-test.js

```diff
@@ -9,10 +9,10 @@ test('qunit-dom assertions are available', function(assert) {
   assert.ok(assert.dom, 'assert.dom is available');
   assert.ok(assert.dom('.foo').includesText, 'assert.dom(...).includesText is available');
 
-  assert.dom('#qunit').doesNotExist('rootElement is set to #ember-testing-container');
-
   visit('/');
   andThen(() => {
+    assert.dom('#qunit').doesNotExist('rootElement is set to #ember-testing-container');
```

Comment:
Had to move this, because the `.ember-view` seems to be rendered after the `visit` call. Before that `document.querySelector('#ember-testing > .ember-view')` will return `null`...

Reply:
Shouldn't be needed with the proposal below.